charts/scrutiny/charts/collector/values.yaml

```diff
@@ -0,0 +1,61 @@
+# Default values for collector.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+env:
+  COLLECTOR_API_ENDPOINT: http://scrutiny-web:8080
+
+image:
+  repository: ghcr.io/analogj/scrutiny
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: master-collector
```

Comment:
we should probably lock this tag to a specific version (that matches the appVersion in the Chart.yaml)

[v0.5.0-web](https://github.com/AnalogJ/scrutiny/pkgs/container/scrutiny/32343361?tag=v0.5.0-web)

Reply:
LGTM, if you agree with keeping the `appVersion` we can make the template default to `{{ .Chart.appVersion }}-web`.